android/res/values-v27/themes.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <style name="MwmTheme.MainActivity">
+    <item name="android:colorPrimaryDark">@android:color/black</item>
+    <item name="android:windowBackground">@null</item>
+    <item name="android:windowTranslucentNavigation">true</item>
+    <item name="android:windowLayoutInDisplayCutoutMode">shortEdges</item>
+  </style>
+  <style name="MwmTheme.Night.MainActivity">
```

Comment:
Does it make sense to mention this file in the corresponding sections of the main themes.xml, so if anyone decides to update it, should also check this file?